modules/ppcp-api-client/src/Endpoint/OrderEndpoint.php

```diff
@@ -572,4 +576,20 @@ public function patch_order_with( Order $order_to_update, Order $order_to_compar
 		$new_order = $this->order( $order_to_update->id() );
 		return $new_order;
 	}
+
+	/**
+	 * Checks if items contains shipping.
+	 *
+	 * @param array $items The items.
+	 * @return bool Whether items contains shipping or not.
+	 */
+	private function items_without_shipping( array $items ): bool {
```

Comment:
This name sounds like it returns the items. So it should either do that or be renamed into something like `has_items_without_shipping`.